HelloApi/HelloApi/HelloApi.csproj

```diff
@@ -161,6 +164,9 @@
       <DependentUpon>Global.asax</DependentUpon>
     </Compile>
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Filters\EventFilter.cs" />
+    <Compile Include="Loggers\IEventLogger.cs" />
+    <Compile Include="Loggers\DataDogStatsdEventLogger.cs" />
```

Comment:
Maybe it's been a bit since I've been in the c# world - do you need to specify every file here? or is there some kind of `*.cs` convention you can use?

Reply:
No idea. I haven't worked in visual studio in a while, this was managed for me by the program itself :\